authorization/auth_cloudSolutionProvider.md

```diff
@@ -0,0 +1,62 @@
+
+> **Note:** This topic applies **only** to Microsoft Cloud Solution Provider (CSP) application developers. The <a href="https://partner.microsoft.com/en-US/cloud-solution-provider" target="_newtab">Microsoft Cloud Solution Provider (CSP)</a> program enables Microsoft’s partners to resell and manage Microsoft Online services to customers. 
+
+**Important**: Calling Microsoft Graph from a CSP application is only supported for directory resources (such as **user**, **group**,**device**, **organization**) and [Intune](..\beta\resources\intune_graph_overview) resources.
+
+## What is a Partner Managed Application?
+The CSP program enables Microsoft’s partners to resell and manage Microsoft Online services such as Office 365, Azure and CRM Online) to customers. Management of customer services is done through Delegated Admin Privileges (DAP) which enables designated partner users and agents to access and configure their customers’ environments.
+
+Additionally, as a partner developer, you can build a Partner Managed App (PMA), also known as a *pre-consented* app, to manage your customers' Microsoft services. PMAs are often termed *pre-consented* apps because all your customers are automatically pre-consented for your partner managed apps. This means when a user from one of your customer tenants uses one of your partner managed apps, the user can use it without seeing any consent. And with the DAP model, your partner agents can access customers through your partner managed application.
+
+## How to set-up a Partner Managed Application
+An application is viewed as *Partner Managed* when it is granted elevated permissions to access your customers' data. Note that partner managed apps can *only* be configured on Partner tenants.
+
+The initial steps required here follow most of the same steps used to register and configure a multi-tenant application:
+
+1. Register your application in your Partner tenant. You can use the <a href="https://portal.azure.com" target="_newtab">Azure Portal</a> for this. To function as a PMA, an application must be configured as a multi-tenant app.
+
+2. Configure your multi-tenant app with the *required permissions* it needs using a least privileged approach.
+
+3. Finally grant your Partner Managed App those configured permissions for all your customers. You can so this by adding the **servicePrincipal** that represents the app to the AdminAgents group in your Partner tenant, using Azure AD powershell V2. You can download and install Azure AD PowerShell V2 from <a href="https://www.powershellgallery.com/packages/AzureAD" target="_newtab">here</a>.  Follow these steps to find the AdminAgents group, the **servicePrincipal** and add it to the group.
+
+    1. Open a PowerShell session and connect to your partner tenant by entering your admin credentials into the sign-in window.
+    ```code
+    > Connect-AzureAd
+    ```
+    2. Next find the group that represents the AdminAgents. 
```

Comment:
We don't need to include "Next", since it's a numbered list of steps; just start step 2 with "Find...".